### DISCUSSION TITLE
Getting 500 Server Error  with Google Sheets Refresh Link

### DISCUSSION THREAD

**Discussion by ghostname3** [code4recovery/12-step-meeting-list#1385] [Q&A] [answered]
Is this on my end?

<img width="1415" alt="Screenshot 2024-03-10 at 11 40 00 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/14844601/5b3a1222-173b-44c9-af4f-98e1b17a4dba">

**Comment by joshreisner** [comment]
hm, what's your google sheet URL? based on the screenshot i'd expect it to be https://docs.google.com/spreadsheets/d/1UZVBgkxi0A670Zu4aS8pZ7CkymyjQkRVIS906AmwHJM/edit but that doesn't appear to exist

**Reply by joshreisner** [reply]
@tamekyle could i get the google sheet URL on google docs? thanks

**Reply by ghostname3** [reply]
https://docs.google.com/spreadsheets/d/1UZVBgkxi0A670Zu4aS8pZ7CkymyjQkRVlS9O6AmwHJM/edit#gid=0

**Reply by joshreisner** [reply, marked as answer]
perfect, thanks! should be fixed now ✅

**Reply by ghostname3** [reply]
Fixed! You are a true legend. Thank you.

**Comment by ghostname3** [comment]
Yeah our sheet url is https://sheets.code4recovery.org/1UZVBgkxi0A670Zu4aS8pZ7CkymyjQkRVlS9O6AmwHJM

Strange that this wouldn't exist. We've been using it for about a year now.. maybe something got crossed up on our end. Any help to fix is greatly appreciated.

We're operating snocoaa.org
